core/templates/components/question-directives/questions-list/questions-list.component.spec.ts

```diff
@@ -772,41 +772,49 @@ describe('Questions List Component', () => {
     let questionId = 'qId';
     let skillDescription = 'Skill Description';
 
-    it('should display warning when user does not have rights to delete' +
```

Comment:
This test is removed because now checking user rights is a part of another component (confirmation modal)